handler.go

```diff
@@ -1480,3 +1628,33 @@ func errorString(err error) string {
 	}
 	return err.Error()
 }
+
```

Comment:
I'd like to see tests specifically for these query functions in handler_internal_tests.go
Also we should probably consistently use use these methods for all endpoints with query args.